lib/AST/CMakeLists.txt

```diff
@@ -62,40 +62,22 @@ add_swift_library(swiftAST STATIC
   TypeWalker.cpp
   USRGeneration.cpp
 
-  LINK_LIBRARIES
-    swiftMarkup
-    swiftBasic
-    swiftSyntax
-
-  INTERFACE_LINK_LIBRARIES
```

Comment:
I see these Clang dependencies are no longer declared, does the use of `add_llvm_library` resolve the need for these explicitly stated?

Reply:
They are, I just reduced them down to the actual dependencies that I believe they were intended to be.  See below with the `target_link_libraries`.